.travis.yml

```diff
@@ -14,6 +14,8 @@ env:
 matrix:
   fast_finish: true
   include:
+    - php: 7.1
+      env: DEPENDENCIES=beta
```

Comment:
What about also adding an explicit job for 3.4 to have a dedicated job for the next LTS release?

Reply:
That's a good idea: I've re-ordered and restructured things. I think it makes sense to test each LTS release, and then also the latest release. Hopefully I've just done that :)